applications/notifications/consumer.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import, unicode_literals
+
+import threading
+
+from applications.notifications import controllers as notifications
+from django.conf import settings
+from json import loads
+
+
+# Kafka consumers
```

Comment:
Could you add a pydoc for this file? While I understand what is going on here thanks to context, somebody new would have to read through this code to understand what is happening.